Mathlib/Algebra/CharP/Basic.lean

```diff
@@ -484,6 +484,10 @@ theorem ringChar_ne_zero_of_finite [Finite R] : ringChar R ≠ 0 :=
   char_ne_zero_of_finite R (ringChar R)
 #align char_p.ring_char_ne_zero_of_finite CharP.ringChar_ne_zero_of_finite
 
+theorem ringChar_zero_iff_CharZero [NonAssocRing F] : ringChar F = 0 ↔ CharZero F := by
+  rw [ringChar.eq_iff]
+  exact ⟨fun _ ↦ charP_to_charZero F, fun _ ↦ ofCharZero F⟩
```

Comment:
How about adding:
```lean
theorem charP_zero_iff_charZero (R : Type*) [AddGroupWithOne R] : CharP R 0 ↔ CharZero R :=
  ⟨fun _ ↦ charP_to_charZero R, fun _ ↦ ofCharZero R⟩
```
just underneath `charP_to_charZero` (above) and then changing this to:
```suggestion
theorem ringChar_eq_zero_iff : ringChar R = 0 ↔ CharZero R := by
  rw [ringChar.eq_iff, charP_zero_iff_charZero]
```

Reply:
I like this suggestion and I just pushed it!

Note that this (or rather, probably the autoImplicit change) made me realize that I had used `F` instead of `R` unknowingly.